crates/pallets/starknet/src/lib.rs

```diff
@@ -345,6 +362,9 @@ pub mod pallet {
         /// The address of the fee token.
         /// Must be set to the address of the fee token ERC20 contract.
         pub fee_token_address: ContractAddress,
+        /// Chain Id, this must be set in the genesis file
+        /// The default value will be MADARA custom chain id
```

Comment:
Is default `SN_GOERLI_TESNET` or `MADARA`? See L332

Reply:
it was `SN_GOERLI`
can we change this to MADARA instead of SN_GOERLI by default? ArgentX has been having issues when the chain_id is SN_GOERLI. they're on a fix but I am not sure when will it be ready by